kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8/virtualkafkaclusters.kroxylicious.io-v1.yml

```diff
@@ -159,6 +159,62 @@ spec:
                                   Supported formats for the bundle are: PEM, PKCS#12 and JKS.
                                 type: string
+                          protocols:
+                            description: |
+                              The allowed or denied TLS protocols.
+                              Either `allowed` or `denied` can be given.
+                              The supported protocols are those supported by the proxy JVM.
+                            type: object
+                            properties:
+                              allowed:
+                                description: |
+                                  The allowed TLS protocols, ordered by preference.
+                                  If not specified this defaults to the TLS protocols of the proxy JVM's default SSL context.
+                                type: array
+                                items:
+                                  type: string
+                              denied:
+                                description: |
+                                  The denied TLS protocols.
+                                  If not specified this defaults to the empty list.
+                                type: array
+                                items:
+                                  type: string
+                                default: []
+                          cipherSuites:
+                            description: |
+                              The allowed or denied cipher suites.
+                              Either `allowed` or `denied` can be given.
+                              The supported cipher suites are those supported by the proxy JVM.
+                            type: object
+                            properties:
+                              allowed:
+                                description: |
+                                  The allowed cipher suites.
+                                  If not specified a default list of ciphers, which depends on the runtime and enabled protocols, will be used.
+                                type: array
+                                items:
+                                  type: string
+                              denied:
+                                description: |
+                                  The denied cipher suites.
+                                  If not specified this will default to the empty list.
+                                type: array
+                                items:
+                                  type: string
+                                default: []
+                          options:
+                            description: |
+                              Options that control TLS server negotiation
+                            type: object
+                            properties:
+                              clientAuth:
```

Comment:
* Can we call this `clientAuthentication`, since 'auth' is an abbreviation both for authentication and authorization?
* I know this is nested in an `tls` property, but I can imagine if we added schema for authentication done by the proxy it could be confusing to have multiple authentication objects. Maybe we can call it `tlsClientAuthentication` to make super-clear that this is just about mtls.